cms/src/components/content/EditContentForm.vue

```diff
@@ -350,6 +389,76 @@ const checkIfDirty = () => {
                     </LInput>
+                    <LInput
+                        name="expiryDate"
+                        label="Expiry date"
+                        class="w-1/2"
+                        type="datetime-local"
+                        :disabled="!canTranslateContent"
+                    >
+                        <LButton
+                            type="button"
+                            variant="secondary"
+                            @click="() => calculateExpirationDate('1 Week')"
+                        >
+                            1W
+                        </LButton>
+                        <LButton
+                            type="button"
+                            variant="secondary"
+                            class="ml-1"
+                            @click="() => calculateExpirationDate('2 Weeks')"
+                        >
+                            2W
+                        </LButton>
+                        <LButton
+                            type="button"
+                            variant="secondary"
+                            class="ml-1"
+                            @click="() => calculateExpirationDate('1 Month')"
+                        >
+                            1M
+                        </LButton>
+                        <LButton
+                            type="button"
+                            variant="secondary"
+                            class="ml-1"
+                            @click="() => calculateExpirationDate('2 Months')"
+                        >
+                            2M
+                        </LButton>
+                        <LButton
+                            type="button"
+                            variant="secondary"
+                            class="ml-1"
+                            @click="() => calculateExpirationDate('3 Months')"
+                        >
+                            3M
+                        </LButton>
+                        <LButton
+                            type="button"
+                            variant="secondary"
+                            class="ml-1"
+                            @click="() => calculateExpirationDate('6 Months')"
+                        >
+                            6M
+                        </LButton>
+                        <LButton
+                            type="button"
+                            variant="secondary"
+                            class="ml-1"
+                            @click="() => calculateExpirationDate('1 Year')"
+                        >
+                            1Y
+                        </LButton>
+                        <LButton
+                            type="button"
+                            variant="secondary"
+                            :icon="BackspaceIcon"
+                            class="ml-1 pt-2"
+                            @click="() => calculateExpirationDate('')"
```

Comment:
Clicking this button doesn't actually reset the value, it just sets it to the current date (it also triggers the "unknown duration" warning from the `calculateExpirationDate` function). Instead I would expect it to totally clear the expiry date, which probably needs a different method like `clearExpirationDate`.